Scripts/restore_fnt.sh

```diff
@@ -18,6 +18,11 @@ do
     tgt=`echo $lst | awk -F '|' '{print $2}'`
     tgt=`eval "echo $tgt"`
 
+    if [[ "${tgt}" =~ /usr/share/ && -d /run/current-system/sw/share/ ]]; then
```

Comment:
Why do you search for that directory precisely instead of just searching anything Nixy, like /nix/store, or even just a `uname`? Is that directory precisely of any use in that evaluation?

Reply:
Honestly I just wasn't aware of any others besides NixOS.

Reply:
> Honestly I just wasn't aware of any others besides NixOS.

Nevermind actually you use that share directory as a substitute in the script, hence why you search for it precisely instead of only looking for NixOS in itself